data/json/vehicle_parts.json

```diff
@@ -1893,592 +2052,1863 @@
             {"item": "plastic_chunk", "count": [1, 2]}
         ]
     },
-    {
-        "type" : "vehicle_part",
-        "id" : "jumper_cable_heavy",
-        "name" : "heavy-duty cable",
-        "symbol" : "{",
-        "color" : "yellow",
-        "broken_symbol" : "*",
-        "broken_color" : "dark_gray",
-        "damage_modifier" : 10,
-        "durability" : 120,
-        "epower" : 5,
-        "//" : "Epower for POWER_TRANSFER stuff is how much percentage-wise loss there is in transmission",
-        "item" : "jumper_cable_heavy",
-        "requirements": {
-          "removal": {
-            "time": 500
-          }
-        },
-        "flags" : ["NOINSTALL", "UNMOUNT_ON_DAMAGE", "UNMOUNT_ON_MOVE", "POWER_TRANSFER"],
-        "breaks_into" : [
-            {"item": "wire", "count": [4, 8]},
-            {"item": "plastic_chunk", "count": [4, 8]}
-        ]
+    "flags": [ "ROOF", "FOLDABLE" ],
```

Comment:
Broken syntax here and in a couple of lines below.

Reply:
Yeah, something went weird on that conflict merge. I'm tempted to just close this PR and re-open it; the commits are way too messy. It was giving me issues with astyle, but when I fixed that, it made vehicle_parts.json have a huge diff change.